src/proto/grpc/testing/stats.proto

```diff
@@ -41,6 +41,12 @@ message ServerStats {
   // change in server time (in seconds) used by the server process and all
   // threads since last reset
   double time_system = 3;
+
+  // Number of requests received
+  uint64 reqs_received = 4;
```

Comment:
Are we referring to RPCs here or messages? In the streaming cases it's the no. of messages that matter.

If we're overloading the meaning here then the doc needs to be improved